src/main/kotlin/eu/kevin/api/services/general/GeneralClient.kt

```diff
@@ -0,0 +1,45 @@
+package eu.kevin.api.services.general
+
+import eu.kevin.api.Endpoint
+import eu.kevin.api.models.ResponseArray
+import eu.kevin.api.models.general.bank.BankResponse
+import eu.kevin.api.models.general.getProjectSettings.GetProjectSettingsResponse
+import io.ktor.client.*
+import io.ktor.client.request.*
+
+class GeneralClient internal constructor(
+    private val httpClient: HttpClient
+) {
+    suspend fun getSupportedCountries(): List<String> =
+        httpClient.get<ResponseArray<String>>(
+            path = Endpoint.Path.General.getSupportedCountries()
+        ).data
+
+    @JvmOverloads
```

Comment:
why here we have `@JvmOverloads` annotation, but we don't have it near the other methods?

Reply:
Because this method has an argument with a default value. In such case, @JvmOverloads generates two separate methods to call from Java (since Java does not have default arguments) - one with an argument and one without one.

Reply:
got it